assignments/lambda-classes.js

```diff
@@ -1 +1,181 @@
 // CODE here for your Lambda Classes
+
+//Person
+class Person {
+	constructor(obj) {
+		this.name = obj.name;
+		this.age = obj.age;
+		this.location = obj.location;
+		this.gender = obj.gender;
+	}
+
+	speak() {
+		console.log(`Hello my ${this.name} is Fred, I am from ${this.location}`);
+	}
+}//Person
+
+//Instructor
+class Instructor extends Person {
+	constructor(obj) {
+		super(obj);
+		this.specialty = obj.specialty;
+		this.favLanguage = obj.favLanguage;
+		this.catchPhrase = obj.catchPhrase;
+	}
+
+	demo(subject) {
+		console.log(`Today we are learning about ${subject}.`);
+	}
+
+	grade(student, subject) {
+		console.log(`${student.name} receives a perfect score on ${subject}`)
+	}
+
+	changeGrade(student) {
+		student.grade += Math.floor(Math.random()*20)-10;
+		console.log(`${this.name} changed ${student.name}'s grade to ${student.grade}`);
+	}
+}//Instructor
```

Comment:
Great job with the stretch and also with these comments! They make your code much cleaner and more readable! Keep it up!